Data/future/cbot/margins/EH.csv

```diff
@@ -1,61 +1,194 @@
 date,initial,maintenance
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv12-019.html
 20120117,4725,3500
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv12-221.html
 20120529,4050,3000
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv12-328.html
 20120806,4455,3300
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv13-168.html
+20130102,3630,3300
+20130212,3080,2800
 20130409,5130,3800
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv13-594.html
+20130410,4180,3800
+20131202,2970,2700
+20131204,4180,3800
```

Comment:
Seems like some of these old values' initial margins are not 110% of the maintenance.